src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" >
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
you need to add two more modificators like they say in redme:

Implement start-day modifier for the calendar with mon, tue, wed, thu, fri, sat and sun values
Use [@each](https://sass-lang.com/documentation/at-rules/control/each) to create all the modifiers
The month should start at the correct column (Monday is the 1st, Friday is the 5th)
You can just add correct margin-left for the first day
Add a modifier month-length for the calendar with values 28, 29, 30 and 31 (use @for)
It sets the last day to show (use [nth-child](https://css-tricks.com/how-nth-child-works/))